geomstats/geometry/special_orthogonal.py

```diff
@@ -112,6 +120,101 @@ def random_uniform(self, n_samples=1, tol=1e-6):
         rotation_mat, _ = gs.linalg.qr(random_mat)
         return rotation_mat
 
+    def rotation_from_angle(self, theta):
+        """Create 2D rotation matrices from scalar angles.
+
+        Parameters
+        ----------
+        theta : float
+            Rotation angle.
+
+        Returns
+        -------
+        mat : array-like, shape = [2, 2],
+            2D Rotation matrix.
+        """
+        if self.n != 2:
+            raise NotImplementedError
+        c = gs.cos(theta)
+        s = gs.sin(theta)
+        mat_first_line = gs.transpose(gs.array([c, -s]))
+        mat_second_line = gs.transpose(gs.array([s, c]))
+        mat_first_line = gs.expand_dims(
+            mat_first_line, -2)
+        mat_second_line = gs.expand_dims(mat_second_line, -2)
+        mat = gs.concatenate(
+            [mat_first_line, mat_second_line], axis=-2)
+        return mat
+
+    def angle_of_rot2(self, point):
```

Comment:
likewise, this should be in `vector_from_rotation_matrix` for consistency

Reply:
What do you think about having `n=2` specfic methods in `SO(n)`? Is there an alternative?